### PR TITLE
Luca

### PR DESCRIPTION
**Pull request by LucaSain**
# MVP PR
## Why?
This is the first pull request had it had to be made by someone.

## When?
Today, 6 December, 17:37

## What?
Just accept the god damn PR @TudorZgimbau 

✨ ✨ Looks good 👍 ✨ ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
-# Project-Cloud4
-The group's presentation website
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (TudorZgimbau) [inline README.md]: Please switch from the default Next.js README

File: components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          </div>
+        </label>
+        <Link href="/">
+          <button className="btn btn-ghost text-xl font-mono ">
+            Project Cloud4
+          </button>
+        </Link>
+      </div>
+      <div>
+        <ul className="menu menu-horizontal  ">
```
Comment (TudorZgimbau) [inline components/Navbar.js:29]: Use array and fix styling.

File: components/two_thingy/twowrap.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Two from "two.js";
+import { useEffect } from "react";
+
+export default function TwoWrapper() {
+  let two = new Two({
+    autostart: true,
+    fullscreen: false,
+  });
+  let twoRef = React.createRef();
```
Comment (TudorZgimbau) [inline components/two_thingy/twowrap.js]: Either import everything from React (*) or limit to the needed elements. I suggest the second for performance.

File: components/two_thingy/twowrap.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  useEffect(() => {
+    two.appendTo(twoRef.current);
+    two.update();
+    var amount = 70;
```
Comment (TudorZgimbau) [inline components/two_thingy/twowrap.js]: Let instead of var

File: pages/_app.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import '../styles/globals.css'
```
Comment (TudorZgimbau) [inline pages/_app.js]: Let's remove separate css files if possible.

File: pages/about.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+          <div className="basis-1/2 flex justify-center items-center flex-col">
+            <div className="flex flex-col">
+              <h1 className=" text-7xl tracking-widest font-bold text-primary ml-3 flex flex-col  ">
+                About Us
+              </h1>
+              <h2 className="text-4xl text-accent tracking-widest font-bold ml-3">
+                We are stuff
+              </h2>
+              <h3 className="text-3xl text-accent tracking-widest font-bold  ml-3">
+                And you are not xd
```
Comment (TudorZgimbau) [inline pages/about.js]: Don't!

File: pages/about.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  return (
+    <div className="bg-[url('/bg.png')] bg-cover">
+      <div className="h-screen w-screen flex flex-col mt-10">
+        <div className="flex flex-row items-center ">
+          <div className="basis-1/2 flex justify-center items-center flex-col">
+            <div className="flex flex-col">
+              <h1 className=" text-7xl tracking-widest font-bold text-primary ml-3 flex flex-col  ">
+                About Us
+              </h1>
+              <h2 className="text-4xl text-accent tracking-widest font-bold ml-3">
```
Comment (TudorZgimbau) [inline pages/about.js]: Use array if possible.

File: pages/about.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+                We are stuff
+              </h2>
+              <h3 className="text-3xl text-accent tracking-widest font-bold  ml-3">
```
Comment (TudorZgimbau) [inline pages/about.js]: Please style using prettier's default rules after the last save(one extra space).

File: pages/api/hello.js [hunk 9]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default function handler(req, res) {
+  res.status(200).json({ name: 'John Doe' })
```
Comment (TudorZgimbau) [inline pages/api/hello.js]: Not necessary, better remove.

File: pages/contact.js [hunk 10]
```diff
@@ -0,0 +1,121 @@
+import Navbar from "../components/NavBar";
+
+function Footer_() {
```
Comment (TudorZgimbau) [inline pages/contact.js]: Please avoid _ in components names.

File: pages/contact.js [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+  const Members = [
```
Comment (TudorZgimbau) [inline pages/contact.js]: camelCase unless better reasons not to.

File: pages/contact.js [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+                  <a
+                    className="text-xl font-mono rounded-lg"
+                    href="https://www.instagram.com/project_cloud4/"
+                    target="blanck"
+                  >
+                    Instagram: @project_cloud4 ;
+                  </a>
+                </li>
+                <li>
+                  <a className="text-xl font-mono rounded-lg">Email:</a>
```
Comment (TudorZgimbau) [inline pages/contact.js]: Missing part here, also check if possible to iterate over an array.

File: pages/contact.js [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+              <a className="text-xl font-mono rounded-lg">
+                You can find us on:
+              </a>
+
+              <ul>
+                <li>
+                  <a
+                    className="text-xl font-mono rounded-lg"
+                    href="https://www.instagram.com/project_cloud4/"
+                    target="blanck"
```
Comment (TudorZgimbau) [inline pages/contact.js]: Typo: blanck -> blank.

File: pages/contact.js [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+                  >
+                    GitHub: https://github.com/Project-Cloud4
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <button
+              className="btn btn-primary"
+              href="C:\Users\GEORGIANA\abrocadabra\src\us.png"
```
Comment (TudorZgimbau) [inline pages/contact.js]: Fix local paths.

File: pages/index.js [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+          <Image fill objectFit="cover" src="/logopc4.png" />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default function Home() {
+  return (
+    <div>
```
Comment (TudorZgimbau) [inline pages/index.js:62]: I would prefer a React fragment for simplicity.

File: pages/projects.js [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+      title: "Equanimity",
+      phrase: "Learning, the true way",
+      image: "/samsung.png",
+      button: "check it now",
+      linkk: "https://sam-eq.netlify.app",
+    },
+  ];
+
+  return (
+    <div className="h-screen  bg-[url('/bg.png')] bg-cover">
```
Comment (TudorZgimbau) [inline pages/projects.js]: Again, styling.

**Review by TudorZgimbau — CHANGES_REQUESTED** [review]

**Comment by TudorZgimbau** [comment]
Also, please update the contact page to the latest version.

File: pages/_app.js (hunk 5 above)
Comment (LucaSain) [reply pages/_app.js]: Importing globals.css is not automatic. It is required to have a imported css file for the tailwind directives.

See [Tailwind Next Guide](https://tailwindcss.com/docs/guides/nextjs) (step 4)

Comment (TudorZgimbau) [reply pages/_app.js]: My bad, sorry.

**Review by TudorZgimbau — APPROVED** [review]

**Pull request merged**